packages/admin/src/common/toolbar/backbutton/ToolbarBackButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { IconButton } from "@material-ui/core";
+import * as React from "react";
+
+import { useStackApi } from "../../../stack";
+import { ToolbarItem } from "../item/ToolbarItem";
+import { useStyles, useToolbarBackButtonThemeProps } from "./ToolbarBackButton.styles";
+
+interface ToolbarBackButtonProps {
+    component?: (goBack?: () => void) => React.ReactNode;
+}
+export function ToolbarBackButton({ component }: ToolbarBackButtonProps) {
+    const stackApi = useStackApi();
+    const classes = useStyles();
+
+    const themeProps = useToolbarBackButtonThemeProps();
+
+    return stackApi && stackApi.breadCrumbs.length > 1 ? (
```

Comment:
I think it should be possible to pass a custom `onClick` prop where the user can handle the go-back-logic himself (optional stack API). Or, if not needed, we should consider naming the Toolbar `StackToolbar`.

Reply:
I wouldn't add onClick, for the same reason as overriding the whole component isn't neccessary. For a custom button you can always create a custom ToolbarItem.

Reply:
removed custom component rendering of `toolbarbackbutton`